recipes/crlibm/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "crlibm" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 48e17981f90d69c6bb0013f68bacbe7a157de864a533d15dd196ca7e98348a35
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - crlibm
+
+about:
+  home: "https://github.com/taschini/pycrlibm"
+  license: "GNU Lesser General Public v2 or later (LGPLv2+)"
```

We prefer SPDX identifiers for consistency
```suggestion
  license: LGPL-2.0-or-later
```

---

Done.